reflex/state.py

```diff
@@ -1259,7 +1271,7 @@ async def _populate_parent_states(self, target_state_cls: Type[BaseState]):
 
         # Fetch all missing parent states and link them up to the common ancestor.
         parent_states_tuple = self._get_parent_states()
-        root_state = parent_states_tuple[-1][1]
+        root_state = self._get_root_state()
```

Comment:
Since parent_states_tuple is already calculated, wouldn't this introduce additional unneeded computation?

Reply:
yes, it does. i can change this line back

Reply:
done